components/membership/membership-persistence-client/src/main/kotlin/net/corda/membership/persistence/client/MembershipPersistenceClient.kt

```diff
@@ -207,23 +206,22 @@ interface MembershipPersistenceClient : Lifecycle {
         ownerX500Name: MemberX500Name,
         ttl: Instant?,
         remarks: String?
-    ): MembershipPersistenceResult<Unit>
+    ): MembershipPersistenceOperation<Unit>
 
     /**
-     * Consumes a pre-auth token provided it exists for the member. If the token was successfully consumed, this method
-     * returns [MembershipPersistenceResult.Success]. Otherwise, it will return [MembershipPersistenceResult.Failure].
+     * Consumes a pre-auth token provided it exists for the member.
```

Comment:
worth preserving this documentation? e.g.
```suggestion
     * Consumes a pre-auth token provided it exists for the member. If the token is successfully consumed, the returned operation will return upon execution [MembershipPersistenceResult.Success]. Otherwise, it will return [MembershipPersistenceResult.Failure].
```